src/_base/harness/config/functions.yml

```diff
@@ -85,6 +85,7 @@ function('filter_local_services', [services]): |
     $filteredService = [];
     foreach ($service as $key => $value) {
       switch ($key) {
+        case 'arguments':
```

Comment:
since this key is quite arbitrary can you instead put it in as:
```
pipeline:
  base:
    mysql:
      attributes: =@('services.mysql.attributes')
```

Reply:
Maybe `command_arguments`?

Reply:
Updated

Reply:
ok we shouldn't need this line with that change

Reply:
Ah-ha, will remove :+1: